es/appendices/glossary.rst

```diff
@@ -1,44 +1,43 @@
-Glossary
+Glosario
 ########
 
-.. glossary::
+.. glosario::
 
-    routing array
-        An array of attributes that are passed to :php:meth:`Router::url()`.
-        They typically look like::
+    Enrutamiento de array
+        Un array de atributos que son pasados a :php:meth:`Router::url()`.
+        Usualmente se tipean de la siguiente manera::
 
             array('controller' => 'posts', 'action' => 'view', 5)
 
-    HTML attributes
-        An array of key => values that are composed into HTML attributes. For example::
+    Atributos HTML
+        Un array de llaves => valores que se compone en atributos HTML. Por ejemplo::
 
-            // Given
+            // Dado
             array('class' => 'my-class', 'target' => '_blank')
 
-            // Would generate
+            // Generara
             class="my-class" target="_blank"
 
-        If an option can be minimized or accepts it's name as the value, then ``true``
-        can be used::
+        Si un atributo acepta su nombre(de atributo) como valor, se puede usar ``true``::
 
-            // Given
+            // Dado
             array('checked' => true)
 
-            // Would generate
+            // Generara
             checked="checked"
 
-    plugin syntax
-        Plugin syntax refers to the dot separated class name indicating classes
-        are part of a plugin. E.g. ``DebugKit.Toolbar`` The plugin is DebugKit,
-        and the class name is Toolbar.
+    Sintaxis de plugin
+        La Sintaxis de plugin se refiere al punto separador del nombre de la clase
+        indicación que la clase es parte de un plugin. Por ej.``DebugKit.Toolbar``.
+        El plugin es DebugKit y el nombre de la clase es Toolbar.
 
-    dot notation
-        Dot notation defines an array path, by separating nested levels with ``.``
-        For example::
+    Notación por puntos
```

Comment:
Glossaries are a bit unique in sphinx, as these terms can be used to generate links. You will also have to update all the places that ``:term:`<term name>``` is used to get the documentation to build correctly.  Another option is to not translate the term names and only translate the term definitions.